packages/odyssey-react-mui/src/TagListContext.tsx

```diff
@@ -0,0 +1,19 @@
+/*!
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { createContext, ElementType } from "react";
+import { ChipProps } from "@mui/material";
+
+export type ChipComponentType = ElementType<ChipProps>;
+export const TagListContext = createContext({
+  chipComponent: "div" as ChipComponentType,
+});
```

Comment:
```suggestion
export const TagListContext = createContext({
  chipComponent: "div",
}: {
  chipComponent: ChipComponentType;
});
```

Explanation _above_.